target_postgres/db_sync.py

```diff
@@ -105,10 +105,20 @@ def flatten_record(d, parent_key=[], sep='__'):
     if not d or not isinstance(d, dict):
         return {}
     items = []
+
     for k, v in d.items():
         new_key = flatten_key(k, parent_key, sep)
-        if isinstance(v, collections.MutableMapping):
+        if isinstance(v, collections.abc.MutableMapping):
```

Comment:
@doigoid can i safely change this?

Reply:
ideally, this would be python backwards compatible, something like:

mutable_class = collections.abc.MutableDict if hasattr(collections, abc) else collections.Mutable Dict